packages/test_track/build.yaml

```diff
@@ -1,6 +1,10 @@
 targets:
   $default:
     builders:
+      source_gen|combining_builder:
+        options:
+          ignore_for_file:
+          - non_constant_identifier_names
```

Comment:
Is this coming from freezed? or json_serializable? something else?

Reply:
The lint violation is coming from the generated code for to/fromJson implementations -- it doesn't like the way they're named. Not sure if that answers your question though 😂